docs/farming-&-staking/farming/advanced-cli/cli-farming-cluster.mdx

```diff
@@ -12,137 +13,420 @@ keywords:
 
-```
-max_payload = 2MB
-```
+:::
+
+:::info Bandwidth Optimization Suggestion
+
+You can reduce LAN bandwidth usage by running the NATS, Controller, Cache, and Plotter components on the same machine. This setup allows most data transfer to occur over the local loopback interface.
+
+:::
+
+## Key Benefits
+
+- **Remote Compute Capability**  
+  Multiple PCs can contribute their CPU and GPU resources for plotting and replotting, without the need to store farm plots locally.
+
+- **Redundancy**  
+  Running multiple computers for each process increases redundancy and system resilience.
+
+- **Bandwidth Efficiency**  
+  Centralized storage of the piece cache helps conserve WAN bandwidth.
+  
+- **Additional Space**  
+  Relocating the piece sync cache to a central location frees up an additional 1% of plot capacity for farming, compared to the Standalone Farmer.
+
+
+## NATS
+
+At the core of Farming Cluster is a third-party software called [NATS](https://nats.io/), which is used for communication between the farming components.
 
-Save this as **`nats.config`** and start the NATS server with Docker:
+The simplest way to install and run NATS is via Docker. 
 
+<details open>
+<summary>Set Up and Configure NATS</summary>
+
+1. Create and save a `nats.config` configuration file with the following:
+
+```bash title="nats.conf"
+max_payload = 2MB
 ```
+2. Start NATS with Docker:
+
+```bash
 docker run \
     --name nats \
     --restart unless-stopped \
     --publish 4222:4222 \
-    --volume ./nats.config:/nats.config:ro \
+    --volume /path/to/nats.config:/nats.config:ro \
     nats -c /nats.config
 ```
 
-### Component Configuration
+</details>
 
-Each of the four components requires a few additional parameters. All need the **`cluster`** specification, a URL to the NATS server, and the specific component being executed.
+---
 
-### Controller
 
-The controller should be the first component to run.  It connects to the node, so the **`node-rpc-url`** must be included.
+## Component Configuration
 
-```
-subspace-farmer cluster --nats-server nats://<NATS_IP>:4222 \
+
+Configuring Cluster Farming is similar to setting up the Standalone Farmer, but each component must be configured according to the cluster setup you want to create. Keep in mind that all components communicate with one another through the NATS server you configured earlier. Below are the essential parameters needed to establish a functional cluster.
+
+<details open>
+<summary>Controller (Manages node communication and orchestrates other components.)</summary>
+
+The first component to start is the Controller. It connects to a single node, and you must specify the node's location on the network.
+
+:::caution Proper Port Forwarding
+
+For your cluster to be reachable, open and forward the default port `30533` to the Controller. If you've changed the port or are using multiple controllers, set it with `--listen-on /ip4/0.0.0.0/tcp/<port>`.
```

Comment:
I'm wondering if we should mention `ip6` here as well since it helps users in some parts of the world where they can get public IPv6 address, but not IPv4.